app/Modules/Newsletter/Model/NewsletterService.php

```diff
@@ -143,24 +136,25 @@ public function sendNewsletters(array $usersNewslettersIds)
 	{
 		$usersNewsletters = $this->userNewsletterModel->getAll()->wherePrimary($usersNewslettersIds)->fetchAll();
 		foreach ($usersNewsletters as $userNewsletter) {
-			$sendGrid = new SendGrid($this->apiKey);
-			$email = new Email;
 
-			$email->addTo($userNewsletter->user->email)
-				->setFrom($userNewsletter->from)
-				->setFromName('Eventigo.cz')
-				->setSubject($userNewsletter->subject)
-				->setCategory('newsletter')
-				->setHtml($userNewsletter->content);
-				//TODO: setText() - we should also send text format
+
+			$to = new Email(null, $userNewsletter->user->email);
+			$from = new Email('Eventigo.cz', $userNewsletter->from);
+
+			$subject = $userNewsletter->subject;
+			$content = new SendGrid\Content('text/plain', $userNewsletter->content); // @todo: or html?
+			//	->setCategory('newsletter') // what is this for
```

Comment:
I see. 

Do you have idea where to put it in SendGrid 5.0?
I found this, but it's very blind without tests: https://github.com/sendgrid/sendgrid-php/blob/master/examples/categories/categories.php#L14

I'd appreciate help with this, never used it.